composer.json

```diff
@@ -27,7 +27,10 @@
     }
   },
   "config": {
-    "bin-dir": "bin"
+    "bin-dir": "bin",
+    "platform": {
+      "php": "5.6.32"
```

Comment:
Is this necessary?

Reply:
This setting tells composer to download dependencies as if it was using PHP 5.6. Otherwise, it relies on the version installed. Without it and with PHP 7.2 installed, I was grabbing a bunch of library version which require PHP 7.1+, making `composer.lock` state not compatible with PHP 5.6 anymore (and also breaking Travis UT).

And the starting point of this is that I couldn't just `composer install` because at its current state I get:
```
friendsofphp/php-cs-fixer 2.3.x-dev requires php ^5.3.6 || >=7.0 <7.2 -> your PHP version (7.2.19) does not satisfy that requirement
```

Reply:
Got it, I think there's a different way to fix this.

First, as I've learned - composer.lock really shouldn't be checked in here. I should fix that.

Second with regards to bundles that aren't compatible with 7.2, I think the right fix is to deprecate support for 5.6 and update the package versions to be 7.1/2/3 compatible.

I suppose I can submit a few changes towards that.

Reply:
Ok, it's going to take a little more time for me to extract the dependencies than I thought. Give me a little time on this.

Reply:
No worries, I'm pulling my fork in the meantime. I think you're right concerning the lock file. At least [this answer](https://stackoverflow.com/a/24247443) is interesting.

And if you drop support EOL PHP versions, it's even better. Let me know if you need me to update this PR.